cmd/make-bm-worker/main.go

```diff
@@ -29,14 +34,35 @@ spec:
   bmc:
     address: {{ .BMCAddress }}
     credentialsName: {{ .Name }}-bmc-secret
+{{- if .MAC }}
+  bootMACAddress: {{ .MAC }}
+{{ end -}}{{- if .WithImage }}
+  userData:
+    namespace: openshift-machine-api
```

Comment:
This setting is also specific to the environments we have downstream.

Reply:
Updated